.gitignore

```diff
@@ -60,6 +60,20 @@ gen
 
 # macOS
 .DS_Store
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.com.apple.timemachine.donotpresent
```

Comment:
some of these files should only be present at the root of the drive, but I'm not complaining unless they cause issues.

Reply:
Hah, funny you should mention that, I have my PowerShell install on the root drive!!

Reply:
Just a couple of them I had to lookup.